packages/alfa-refinement/src/refinement.ts

```diff
@@ -70,7 +70,8 @@ export namespace Refinement {
 
     <T, U extends T, A extends Array<unknown> = []>(
       left: Refinement<T, U, A>,
-      right: Predicate<U, A>
+      right: Predicate<U, A>,
+      ...predicates: Array<Predicate<U, A>>
```

Comment:
This introduces an odd asymmetry with the remaining overloads and the `or()` function:

```ts
// With the proposed changes, this is valid...
and(refinement, ...predicates);

// ...but this isn't:
and(...predicates, refinement);
```

Let's not do that and instead nest `and()` calls which maintains the symmetry:

```ts
// Currently, this is valid...
and(refinement, and(...predicates));

// ...and so is this:
and(and(...predicates), refinement);
```